File: Python/8py 2/multiply_xy.py
"""
|--------------------------------------------------------------------------
| Function 3 - multiplying two numbers
|--------------------------------------------------------------------------
|
"""

def multiply(x,y):
   return x*y


multiply = lambda _, __: _*__


from operator import mul
def multiply(*args):
   return mul(*args)


def multiply(a,b):
   return (a).__mul__(b)


def multiply(*_):
   return _[0]*_[1]


print(multiply(2,3), 6)

File: PHP/8php 2/multiply_xy.php
<?php
/*
|--------------------------------------------------------------------------
| Function 3 - multiplying two numbers
|--------------------------------------------------------------------------
|
*/

function multiply($x, $y) {
   return $x * $y;
}


function multiply(...$n) {
   return array_product($n);
}


function multiply() {
   return array_product(func_get_args());
}


print_r(multiply(2,3) . ' ' . 6);